runner.py

```diff
@@ -321,30 +341,17 @@ def get_main_parser():
     ## create tmp directory and check file exist or not
     from os import path
 
-    if path.exists(f"{args.output}") and args.overwrite == False:
-        raise Exception(f"{args.output} exists")
+    if path.exists(f"{coffeaoutput}") and args.overwrite == False:
+        raise Exception(f"{coffeaoutput} exists")
+
     if args.isArray:
-        ## create the directory
-        if (
-            path.exists(args.output.replace(".coffea", "").replace("hists_", ""))
-            and args.overwrite == False
-        ):
+        outdir = f"arrays_{args.workflow}_{sample_json.rstrip('.json')}"
+        if path.exists("tmp"):
+            os.system("rm -r tmp")
+        os.mkdir("tmp")
+
+        if path.exists(outdir) and args.overwrite == False and args.only is None:
             raise Exception("Directory exists")
```

Comment:
Indeed it's better not to use tmp directory if you have multiple jobs running. create the directory after L354 
```suggestion
        outdir = f"arrays_{args.workflow}_{sample_json.rstrip('.json')}"
        if path.exists(outdir) and args.overwrite == False and args.only is None:
             raise Exception("Directory exists")
        else:
            os.system(f"mkdir -p {outdir}")
```